src/memory.c

```diff
@@ -199,14 +199,20 @@ static uint8_t memory_eeprom_crypt(const uint8_t *write_b, uint8_t *read_b,
     if (!dec) {
         goto err;
     }
-    memcpy(read_b, utils_hex_to_uint8(dec), MEM_PAGE_LEN);
+    if (read_b) {
+        memcpy(read_b, utils_hex_to_uint8(dec), MEM_PAGE_LEN);
```

Comment:
it would be nice if the args of `memory_eeprom_crypt` func were documented. For instance, I see it takes `const uint8_t *write_b` and `uint8_t *read_b` but no length. Looking at the code, I assume the length is always `MEM_PAGE_LEN` but would be nicer to confirm this reading a function doc comment.

Reply:
ACK
Added the comment. Thanks.